buildkite/src/Jobs/Bench/LedgerApplyUnstable.dhall

```diff
@@ -0,0 +1,15 @@
+let LedgerApply = ../../Command/Bench/LedgerApply.dhall
+
+let PipelineMode = ../../Pipeline/Mode.dhall
+
+let Pipeline = ../../Pipeline/Dsl.dhall
+
+in  Pipeline.build
+      ( LedgerApply.pipeline
+          LedgerApply.Spec::{
+          , name = "LedgerApplyUnstable"
+          , label = "Ledger Apply"
+          , key = "ledger-apply"
+          , mode = PipelineMode.Type.Stable
```

Comment:
Is it a mistake? A bit counter-intuitive that the **LedgerApplyUnstable.dhall** file uses `Stable` pipeline mode.

Reply:
Ups. Yes, this is a mistake